Projects/UOContent/Items/Lights/Lantern.cs

```diff
@@ -6,10 +6,12 @@ namespace Server.Items;
 [SerializationGenerator(0, false)]
 public partial class Lantern : BaseEquipableLight
 {
+    public static readonly TimeSpan FullDuration = TimeSpan.FromMinutes(20);
+
     [Constructible]
     public Lantern() : base(0xA25)
     {
-        Duration = Burnout ? TimeSpan.FromMinutes(20) : TimeSpan.Zero;
+        Duration = FullDuration;
```

Comment:
Why change the conditional? I think the `Burnout` if I remember correctly is to have a configurable way for admins to change the way lighting works. 🤔

Reply:
Ah okay, I assumed since it was hard-coded it was being used as a bookmark for things which don't burn out. I'll revert that.